linux-4.9.3/fs/binfmt_elf.c

```diff
@@ -9,40 +9,40 @@
  * Copyright 1993, 1994: Eric Youngdale (ericy@cais.com).
  */
 
-#include <linux/module.h>
```

Comment:
Doesn't this make things hard if you ever need to diff against the upstream kernel (e.g., if you want to see what was changed in S2E land)?

Reply:
When upgrading, you would replay all relevant commits. The formatting ones will be skipped.
Moreover, Linux is moving to clang-format since last year, so any upgrade shouldn't be to complex to handle.
https://github.com/torvalds/linux/commits/master/.clang-format
https://www.kernel.org/doc/html/v4.17/process/clang-format.html

Reply:
Ah ok, fair enough.